test/models/test_artifact_rules.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python
+
+"""
+<Program Name>
+  models/test_artifact_rules.py
+
+<Author>
+  Santiago Torres-Arias <santiago@nyu.edu>
+
+<Started>
+  Nov 19, 2016
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Test artifact rule unpacking.
+
+"""
+
```

Comment:
Yep, we [want this](https://github.com/secure-systems-lab/code-style-guidelines#documentation-strings). But I won't change this everywhere in this PR. This could be candidate for a general code cleanup PR.
> Note that most importantly, the """ that ends a multiline docstring should be on a line by itself, and preferably preceded by a blank line, e.g.: